test/parallel/test-child-process-execfile.js

```diff
@@ -2,10 +2,10 @@
 const common = require('../common');
 const assert = require('assert');
 const execFile = require('child_process').execFile;
-const path = require('path');
 const uv = process.binding('uv');
+const { fixturesPath } = require('../common/fixtures');
 
-const fixture = path.join(common.fixturesDir, 'exit.js');
+const fixture = fixturesPath('exit.js');
```

Comment:
@jasnell taking this test as an example, going forward when looking at this test one has to remember that `fixturesPath` is from common, and `fixture` is defined in this test. Multiply that by 1500 tests and five common modules and things get complex.

More compact syntax seems to me to be favouring writability over readability, I'd rather do the opposite.

I'm fine with `2.`, it's slightly more mental load, but not much, and `fixtures.path()` is no less compact than `fixturesPath()` 😁